sklearn/ensemble/_forest.py

```diff
@@ -324,6 +324,14 @@ def fit(self, X, y, sample_weight=None):
             # [:, np.newaxis] that does not.
             y = np.reshape(y, (-1, 1))
 
+        if self.criterion == "poisson":
+            if np.any(y < 0):
+                raise ValueError("""Some value(s) of y are negative which is \
+                                    not allowed for Poisson regression.""")
```

Comment:
```suggestion
                raise ValueError("Some value(s) of y are negative which is "
                                 "not allowed for Poisson regression.")
```
We usually try to avoid linebreak `\` and triple quote `"""`(except in docstrings).